src/content/docs/distribute/index.mdx

```diff
@@ -3,12 +3,70 @@ title: Distribute
 sidebar:
   order: 0
   label: Overview
-  badge:
-    text: WIP
-    variant: caution
 ---
 
-import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+import { CardGrid, LinkCard, LinkButton } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Tauri provides the tooling you need to distribute your application either to the platform app stores or as platform-specific installers.
+
+## Building
+
+Tauri builds your application directly from its CLI via the `build`, `android build` and `ios build` commands.
+
+<CommandTabs
+  npm="npm run tauri build"
+  yarn="yarn tauri build"
+  pnpm="pnpm tauri build"
+  cargo="cargo tauri build"
+/>
+
+See the [distributing](#distributing) section to learn more about the configuration options available for each bundle
+and how to distribute them to your users.
+
+:::note
+Most platforms requires code signing. See the [signing](#signing) section for more information.
+:::
+
+### Bundling
+
+By default the `build` command automatically bundles your application for the configured formats.
+
+If you need further customization on how the platform bundles are generated, you can split the build and bundle steps:
+
+<CommandTabs
+  npm="npm run tauri build --no-bundle
+# bundle for distribution outside the macOS App Store
+npm run tauri bundle --bundles app,dmg
+# bundle for App Store distribution
+npm run tauri bundle --bundles app --config src-tauri/tauri.appstore.conf.json"
+  yarn="yarn tauri build --no-bundle
+# bundle for distribution outside the macOS App Store
+yarn tauri bundle --bundles app,dmg
+# bundle for App Store distribution
+yarn tauri bundle --bundles app --config src-tauri/tauri.appstore.conf.json"
+  pnpm="pnpm tauri build --no-bundle
+# bundle for distribution outside the macOS App Store
+pnpm tauri bundle --bundles app,dmg
+# bundle for App Store distribution
+pnpm tauri bundle --bundles app --config src-tauri/tauri.appstore.conf.json"
+  cargo="cargo tauri build --no-bundle
+# bundle for distribution outside the macOS App Store
+cargo tauri bundle --bundles app,dmg
+# bundle for App Store distribution
+cargo tauri bundle --bundles app --config src-tauri/tauri.appstore.conf.json"
+/>
+
+## Versioning
+
+Your application version can be defined in the [`tauri.conf.json > version`] configuration option,
+which is the recommended way for managing the app version. If that config value is not set,
+Tauri uses the `package > version` value from your `src-tauri/Cargo.toml` file instead.
+
+:::note
+Some platforms have some limitations and special cases for the version string.
+See the individual distribution documentation pages for more information.
```

Comment:
```suggestion
See the individual distribution documentation pages for more information.
```
idk, this sentence sounds odd to me but I don't want to repeat "platform": 

"See the specific platform documentation pages for more information"

Reply:
looks good